srcpkgs/senpai/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'senpai'
+pkgname=senpai
+version=0.0.20211223
+revision=1
+_gitrev=8091752a67781273944e7a79a803b7a671378313
+wrksrc="${pkgname}-${_gitrev}"
+build_style=go
+go_import_path="git.sr.ht/~taiite/senpai"
+go_package="./cmd/senpai"
+hostmakedepends="scdoc"
+short_desc="Your everyday IRC student"
```

Comment:
It would be nice to mention this is actually an IRC client

Reply:
I agree, that description is very vague. I just took it from the website that was previously up for the project.